website/src/plugins/docusaurus-plugin-segment/package.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
i dont think you need this one, since its an inline plugin, it will see the project dependencies

Reply:
Yes, IDE was complaining, but works even without `package.json`. Thanks!